src/System.DirectoryServices.Protocols/ref/System.DirectoryServices.Protocols.cs

```diff
@@ -640,4 +643,4 @@ public partial class VlvResponseControl : System.DirectoryServices.Protocols.Dir
     public int TargetPosition { get { return default(int); } }
   }
 }
-
\ No newline at end of file
+
```

Comment:
I know this wasn't your change, but might be worth deleting this weird char.

Reply:
Is that not just a normal line break?

Reply:
Maybe. Didn't look locally. Will merge now.